tests/system/Helpers/ArrayHelperTest.php

```diff
@@ -181,4 +251,58 @@ public function deepSearchProvider()
 			],
 		];
 	}
-}
+
+	public function sortByMultipleKeysProvider()
```

Comment:
```suggestion
	public static function sortByMultipleKeysProvider(): iterable
```

You can combine this with your seeder below, so that when you feed the data to your test methods they can accept three arguments: `public function testSomething($input, $sortColumns, $expected)`.

Reply:
Done.